dockerfiles/cpubuilder/build_manylinux_x86_64.Dockerfile

```diff
@@ -11,6 +11,9 @@
 # new upstream versions as needed.
 FROM quay.io/pypa/manylinux_2_28_x86_64@sha256:634656edbdeb07f955667e645762ad218eefe25f0d185fef913221855d610456
 
+# Prepend therock-tools to PATH for all subsequent steps
+ENV PATH="/usr/local/therock-tools/bin:$PATH"
```

Comment:
Should this go at the _end_ of the dockerfile instead of the start, so anything put in therock-tools takes precedence? Other steps may also prepend (as line 20 does immediately after this)

Reply:
I told Erman to put it at the top. I think it is fine at the top, but let's just have one path setting (combine this and the line below).